src/daal4py.cpp

```diff
@@ -851,6 +851,45 @@ void c_train_test_split(data_or_file & orig, data_or_file & train, data_or_file
     daal::data_management::internal::trainTestSplit<int>(origTable, trainTable, testTable, trainIdxTable, testIdxTable);
 }
 
+double c_roc_auc_score(data_or_file & y_true, data_or_file & y_test)
+{
+    const size_t col_true = y_true.table->getNumberOfColumns();
+    const size_t row_true = y_true.table->getNumberOfRows();
+    const size_t col_test = y_test.table->getNumberOfColumns();
+    const size_t row_test = y_test.table->getNumberOfRows();
+    if (row_true != 1 || row_test != 1 || col_true != col_test) {
+        PyErr_SetString(PyExc_RuntimeError, "Unknown shape data");
+        return NULL;
+    }
+
+    auto table_true = get_table(y_true);
+    auto table_test = get_table(y_test);
+    auto type = (*table_test->getDictionary())[0].indexType;
+    if (type == daal::data_management::data_feature_utils::DAAL_FLOAT64 ||
+        type == daal::data_management::data_feature_utils::DAAL_INT64_S ||
+        type == daal::data_management::data_feature_utils::DAAL_INT64_U)
+    {
+        double score;
+        double* predictedRank = new double[col_true];
+        daal::data_management::internal::calculateRankData<double>(predictedRank, table_test, col_true);
```

Comment:
Merge to one function for example

```suggestion
        score = daal::data_management::internal::rocAucScore<double>(table_true, table_test);
```